src/main/java/net/mcreator/element/types/Biome.java

```diff
@@ -35,6 +35,7 @@
 	public final transient int TREES_VANILLA;
 	public final transient int TREES_CUSTOM;
 
+	public String name;

```

Comment:
Add test case for this in https://github.com/MCreator/MCreator/blob/5fa77237a5f1512baf9d1f42dab8cd92a815a998/src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java#L204